### PR TITLE
pull request from UVA pysumma notebook example

### PR DESCRIPTION
**Pull request by DavidChoi76**

### CODE REVIEW

**Comment by DavidChoi76** [comment]
@bartnijssen @arbennett could one of you review this pull request. it consists of one example notebook and supporting files.

File: pysumma/Simulation.py [hunk 1]
```diff
@@ -30,12 +30,19 @@ def __init__(self, filepath):
         self.output_prefix = FileManagerOption('output_prefix', self.filepath)
         self.decision_obj = Decisions(self.setting_path.value + self.decision_path.value)
 
-    def execute(self):
+    def execute(self,     run_suffix):
```
Comment (arbennett) [inline pysumma/Simulation.py]: No need for extra spaces before `run_suffix`

File: pysumma/Simulation.py [hunk 2]
```diff
@@ -30,12 +30,19 @@ def __init__(self, filepath):
...
+        self.run_suffix = run_suffix
         if self.executable == '':
             raise ValueError('No executable defined. Set as "executable" attribute of Simulation')
         else:
             cmd = "{} -p never -s {}       -m {}".format(self.executable, self.run_suffix, self.filepath)
```
Comment (arbennett) [inline pysumma/Simulation.py]: No need for extra spaces before `-m`

File: pysumma/Simulation.py [hunk 3]
```diff
@@ -30,12 +30,19 @@ def __init__(self, filepath):
...
             subprocess.run(cmd, shell=True)
+            out_file_path = 	self.output_path.filepath + \
```
Comment (arbennett) [inline pysumma/Simulation.py]: This is somewhat brittle, but maybe beyond the scope of this pull request.  We should figure out a better way to pick up where output comes from.  I'll open an issue on this topic.  I think it's fine here for now, though.

**Review by arbennett — CHANGES_REQUESTED** [review]
I think it would be good to move the example notebook to a `notebooks` or `examples` directory so that it's easier to expand things out in the future.

**Comment by DavidChoi76** [comment]
@arbennett :  I changed your requests. Sorry. It's too late.

**Review by arbennett — APPROVED** [review]

**Pull request merged**